docs/reference/settings/ml-settings.asciidoc

```diff
@@ -22,15 +22,19 @@ hardware, you must disable {ml} (by setting `xpack.ml.enabled` to `false`).
 
 `node.roles: [ ml ]`::
 (<<static-cluster-setting,Static>>) Set `node.roles` to contain `ml` to identify
-the node as a _{ml} node_ that is capable of running jobs. Every node is a {ml}
-node by default.
+the node as a _{ml} node_. If you want to run {ml} jobs, there must be at least
+one {ml} node in your cluster. 
 +
-If you use the `node.roles` setting, then all required roles must be explicitly
-set. Consult <<modules-node>> to learn more.
+By default, every node is a {ml} node. If you set `node.roles`, however,
+you must explicitly specify all the required roles for the node. To learn more, 
```

Comment:
I think this sentence:

> If you set `node.roles`, however, you must explicitly specify all the required roles for the node.

Would be better in `docs/reference/modules/node.asciidoc`

Reply:
I think it's already covered in the following sentence on that page: "If you set `node.roles`, the node is assigned only the roles you specify."